library/openshift_provision.py

```diff
@@ -1467,15 +1467,25 @@ def provision(self):
         # Perform action on resource
         resource_fqdn = self.resource['kind'] + "." + self.resource['apiVersion'].split("/")[0]
```

Comment:
```suggestion
        if self.resource['apiVersion'].split("/")[0] == "v1":
                resource_fqdn = self.resource['kind']
            else:
                resource_fqdn = self.resource['kind'] + "." + self.resource['apiVersion'].split("/")[0]
```

Refactor the name if it is not pleasing. Then remove other changes?